app/forms/gobierto_admin/gobierto_cms/page_form.rb

```diff
@@ -73,6 +80,8 @@ def collection_class
       end
 
       def save_section_item(id, section, parent)
+        return if section_instance.blank? && page.section.blank?
```

Comment:
should this be an OR?

Reply:
No, in this way the admin can remove the section of a page (in this case the section_instance is blank but the page section is present)

Reply:
Ok, understood!